server/calendar/calendar-router.js

```diff
@@ -0,0 +1,77 @@
+var calendarController = require('./calendar-controller.js');
+var express = require('express');
+var calendarRouter = express.Router();
+var User = require('./User');
+
+
+calendarRouter.get('/', calendarController.signin);
+
+calendarRouter.get('/callback', calendarController.callback);
+
+calendarRouter.post('/checkloggedin', calendarController.isAuthenticated);
+
+calendarRouter.get('/logout', calendarController.logout);
+
+//Temporary Failure route if passport authentication fail
+calendarRouter.get('/failure', function() {
+  console.log('Something went wrong in authenticating with google');
+});
+
+//Temporary route to get all users calendar meta-information
+calendarRouter.get('/allCalendars', function (req, res) {
```

Comment:
won't these handler functions be better organized if we keep all the logic within the controller? it may make our routes files look cleaner also. I organized the employees and organization routers and controllers in that way if you want to compare - what are your thoughts on this?